src/components/ConversationIcon.vue

```diff
@@ -167,10 +179,11 @@ export default {
 		iconClass() {
 			if (this.item.isDummyConversation) {
 				// Prevent a 404 when trying to load an avatar before the conversation data is actually loaded
-				// Also used in new conversation dialog
+				// Also used in new conversation / invitation handler dialog
+				const isFed = this.item.isFederatedConversation && 'icon-conversation-federation'
```

Comment:
Should we consider it a type from now on ?

Reply:
Might complicate things and on the API side